content/docs/rfcs/31/README.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: 31
+title: 31/WAKU2-TRANSPORT
```

Comment:
As discussed in call, I'd embed this into main Waku v2 spec. Not sure it makes sense to have as a separate spec, at least at this stage.

Reply:
Agree - also note that RFC31 is already taken (https://github.com/vacp2p/rfc/pull/465)

Reply:
fixed